CHANGELOG.rst

```diff
@@ -4,6 +4,31 @@ community.aws Release Notes
 
 .. contents:: Topics
 
+v10.0.0
+=======
+
+Release Summary
+---------------
+
+In this major release, the ``connection/aws_ssm`` connection plugin has been fully migrated to the ``amazon.aws`` collection (https://forum.ansible.com/t/aws-ssm-connection-refactoring-plugin-promotion/39930). Playbooks or Inventory using the Fully Qualified Collection Name (FQCN) for this connection plugin should be updated to use ``amazon.aws.aws_ssm``.
+
+Major Changes
+-------------
+
+- community.aws collection - The ``community.aws`` collection has dropped support for ``botocore<1.34.0`` and ``boto3<1.34.0``. Most modules will continue to work with older versions of the AWS SDK, however compatibility with older versions of the SDK is not guaranteed and will not be tested. When using older versions of the SDK a warning will be emitted by Ansible (https://github.com/ansible-collections/amazon.aws/pull/2426).
+
+Breaking Changes / Porting Guide
+--------------------------------
+
+- Support for ``ansible-core<2.17`` has been dropped (https://github.com/ansible-collections/community.aws/pull/2303).
```

Comment:
Keeping this similar to the changelog in https://github.com/ansible-collections/amazon.aws/pull/2601